pkg/rancher-desktop/components/UpdateStatus.vue

```diff
@@ -223,6 +235,18 @@ export default UpdateStatus;
     margin-left: 2em;
     margin-right: 1em;
   }
+
+  .check-updates {
+    display: flex;
+    gap: 10px;
+    align-items: center;
+  }
+
+  .btn-xs {
```

Comment:
That looked too generic, but that's what all the other buttons with limited height do, so I guess that's okay‽